www-apache/mod_jk/mod_jk-1.2.48.ebuild

```diff
@@ -48,6 +39,9 @@ src_compile() {
 }
 
 src_install() {
+	# otherwise installs to /mod_jk.so
+	APACHE_MODULESDIR="/usr/$(get_libdir)/apache2/modules"
```

Comment:
@Polynomial-C is this according to plan? It looks like `apache-module.eclass` is using that variable without having its content guaranteed by `need_apache2`.